CHANGELOG.md

```diff
@@ -1,3 +1,199 @@
+# 5.1.0
+
+
+#### Blocking proxy
+
+Blocking Proxy is a new experimental feature in Protractor 5 and is behind the
+`--useBlockingProxy` or `blockingProxyUrl`. See the
+[lib/config.ts#L100](https://github.com/angular/protractor/blob/master/lib/config.ts#L100).
+Other ways to start blocking proxy include using the
+`--highlightDelay` and `--webDriverLogDir` flags See [lib/config.ts#L501](https://github.com/angular/protractor/blob/master/lib/config.ts#L501).
+
+This adds two options, both of which are implemented with Blocking
+Proxy. `--webDriverLogDir` will create a readable log with timing
+information of webdriver commands in the specified directory.
+
+`--highlightDelay` will pause before clicking on elements or sending keys.
+While paused, the element that's about to be affected will be
+highlighted.
+
+#### Webdriver-manager
+
+Webdriver-manager will now by default grab the latest versions of all binaries
+(standalone, chromedriver, iedriver, gecko driver). Use the
+`--versions.(binary name)` to pin to a specific version. Selenium standalone
+3.0.1 has a bug which prevents it from working with any version of FireFox.
+We have tested version 3.0.0-beta4 and know that it works with
+FireFox 51, and we expect that the 3.0.2 release will also work.
+
+#### Breaking changes summary:
+
+- `browser.getLocationAbsUrl` deprecated
+- Replaced `browser.rootEl` with `browser.angularAppRoot()`
```

Comment:
Note that `browser.rootEl` will still work - so I'd prefer to not list this here.